src/lib/validate-forms.js

```diff
@@ -43,6 +45,13 @@ module.exports = async (projectDir, subDirectory, options={}) => {
   }
 
   const instanceProvided = environment.apiUrl;
+  let apiVersion;
+  if(instanceProvided) {
+    const version = await getApiVersion();
+    if (semver.valid(version)) {
```

Comment:
is this semver validation here because the api version can be a regular branch name and we don't want to expose it? 
The [getApiVersion](https://github.com/medic/cht-conf/blob/main/src/lib/get-api-version.js) lib isn't very extensible right now, but what do you think if it offers the option of return a semver valid api version and the raw api version? 

If we want to use `getApiVersion` again somewhere else in the code, then we will need to add this validation in the new code as well.

Reply:
If you decide to leave it here, can you take it out as a function? It'll make this function straight forward and still understandable.

Reply:
Okay, so I double-checked to verify that, yes, if you install a branch via admin/horticulturist then the branch name will come back as the value for `deploy_info.version`.  Currently the only times the version is used in cht-conf, we want to check that is is valid semver (since we will be comparing it against other versions).

So, I think it makes sense to include the `semver.valid` check in the `get-api-version` lib. But, in the future, there might be a use case that only wants to include the version in a report or something (and so it does not need to be valid semver).  With that in mind, I added a new `getValidApiVersion` function to `get-api-version` to make it easier to expose the unvalidated version in the future.